index.html

```diff
@@ -8319,8 +8350,8 @@ <h3>Introduction</h3>
 <h2>Archive sites</h2>
 
 <p>This specification can be found at
-<a href="http://www.w3.org/TR/2003/REC-PNG-20031110/index.html"
-><code>http://www.w3.org/TR/2003/REC-PNG-20031110/index.html</code></a>.</p>
+<a href="http://www.w3.org/TR/2022/WD-PNG-20221025/"
```

Comment:
Same here. This link seems to work even though it is not Oct 25th yet. Is this an estimated date?